### PR TITLE
[OCPBUGS-6728]: Delete node objects during etcd restore to prevent OVN-Kubernetes from starting its DBs as clustered

### PR DESCRIPTION
**Pull request by tmalove**
[OCPBUGS-6728](https://issues.redhat.com/browse/OCPBUGS-6728)

Version(s): 4.11+

Link to docs preview: http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6728-tlove/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state  (Step 12)

QE review:
- [ x] QE has approved this change.

### CODE REVIEW

**Comment by tmalove** [comment]
@palonsoro  Will you review this prerequisite update for the minimum number of controllers? Thanks!

**Comment by tmalove** [comment]
@geliu2016 Will you review this prerequisite update? Thanks!

**Comment by palonsoro** [comment]
This is not an acceptable solution at all, so I am not giving an OK to this.

Both here[1] and here[2] we say that "This can be used to recover from the following situations: (...) The cluster has lost the majority of control plane hosts (quorum loss).".

We cannot go and all of a sudden say that now we require the controller nodes being active, because:
- From a product perspective, it is not serious to not be able to recover from a 2 controller node fall.
- Apparently, this used to work also without this requirement, at least when OVN-Kubernetes was not in the equation.
- The previous point means that we cannot consider being able to recover from 2 controller nodes a new feature at all, but we must consider this not working with OVN-Kubernetes a bug or even a regression.

In short, the solution is to make a procedure that works without this requirement, not just add requirements as we see problems in our procedures instead of fixing them.

[1] - https://docs.openshift.com/container-platform/4.11/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.html
[2] - https://docs.openshift.com/container-platform/4.11/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

**Comment by tmalove** [comment]
@palonsoro @geliu2016 Will you verify the new step 12 that deletes the node? The preview is updated. Thanks!

**Comment by palonsoro** [comment]
The preview looks good to me.

**Comment by palonsoro** [comment]
/retitle Delete node objects during etcd restore to prevent OVN-Kubernetes from starting its DBs as clustered

**Comment by palonsoro** [comment]
@tmalove the PR had a title according with the first changes and it was no longer correct. I have changed it but feel free to change it again if you feel it should be different.

**Comment by bmcelvee** [comment]
LGTM!

Just a note, our guidelines say that:

> If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:
> 
> <Step description> by running the following command:
> 
> <Step description> by entering the following command:
> 
> <Step description>, run the following command:
> 
> <Step description>, enter the following command:

But what you have here is consistent with other steps in the procedure. Changing it would be a larger effort and can be done later, I think.

**Comment by tmalove** [comment]
> LGTM!
> 
> Just a note, our guidelines say that:
> 
> > If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:
> >  by running the following command:
> >  by entering the following command:
> > , run the following command:
> > , enter the following command:
> 
> But what you have here is consistent with other steps in the procedure. Changing it would be a larger effort and can be done later, I think.

Thanks @bmcelvee ! After this is resolved for the customer, I will take a look at the entire procedure and update them for compliance.

**Comment by ousleyp** [comment]
LGTM; merging now!
